{{cookiecutter.app_name}}/app.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
The cookiecutter template has a [post gen hook](https://github.com/cookiecutter-flask/cookiecutter-flask/blob/master/hooks/post_gen_project.py) that runs after the project is generated. Given that `app.json` and `Procfile` are only relevant for Heroku deployments, could you add to this hook such that these files are deleted from the generated project if the user doesn't choose a Heroku deployment?

Reply:
Sure, great point. I like this idea a lot.